tests/phpunit/unit-tests/forms/class-llms-test-form-handler.php

```diff
@@ -324,11 +325,85 @@ public function test_submit_success() {
 		wp_set_current_user( $ret );
 		$args['first_name'] = 'Maude';
 		$args['display_name'] = $user->display_name;
+		// Current password is required when updating the password.
+		$args['password_current'] = $args['password'];
 		$this->assertSame( $ret, $this->handler->submit( $args, 'account' ) );
 		$this->assertEquals( $args['first_name'], $user->first_name );
 
 	}
 
+
+	/**
+	 * Test submit password change without providing, or with wrong current password
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_submit_password_update_wrong_current_password() {
```

Comment:
I think this test and the next test have transposed names/descriptions

Reply:
you're right :D